src/connector/src/sink/encoder/mod.rs

```diff
@@ -82,6 +82,16 @@ impl<T> SerTo<T> for T {
     }
 }
 
+pub trait QueryField {
+    fn get_field(&self, name: &str) -> Option<String>;
+}
```

Comment:
The reason I left it here is because I think it can be useful for other protocols like Nats or even kafka. To do a dynamic routing, I can move it to the mqtt sink if you think it won't apply to others